packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -313,59 +293,74 @@ export class IndexedDbRemoteDocumentCache implements RemoteDocumentCache {
       .next(() => results);
   }
 
-  getNewDocumentChanges(
-    transaction: PersistenceTransaction
-  ): PersistencePromise<MaybeDocumentMap> {
+  /**
+   * Returns the set of documents that have been updated since the specified read
+   * time.
+   */
+  // PORTING NOTE: This is only used for multi-tab synchronization.
+  getDocumentChanges(
```

Comment:
FWIW, the `getNewDocumentChanges` still makes sense while explicitly passing the `readTime`. That name helps convey that it's a diff of changes.

I don't oppose renaming it to `getDocumentChanges` if that's your preference, but I don't think it needed to change.